ci/pipeline/main.yml.erb

```diff
@@ -167,8 +179,58 @@ jobs:
         rebase: true
         tag_prefix: v
         tag: integer-version/tag-file
-    - put: release-version-semver
-      params: { file: release-version-semver/number }
+    - put: past-releases-version-semver
+      params: { file: past-releases-version-semver/{{vcpi_branch}} }
+
+  - name: promote-candidate-minor #minor-release
+    serial: true
+    plan:
+      - aggregate:
+          - get: source-ci
+          - get: bosh-cpi-artifacts
+            passed: [inspect-candidate]
+          - get: bosh-cpi-src
+            passed: [inspect-candidate]
+          - get: past-releases-version-semver
+            params: { bump: minor }
+      - task: promote
+        file: source-ci/ci/tasks/promote-candidate.yml
+        params:
+          AWS_ACCESS_KEY_ID:     {{s3_vsphere_cpi_blobwriter_access_key}}
```

Comment:
All promote(major, minor, patch) will become separate erb files. All code for these jobs except for their rendering above will not be in this file. It is similar to how you have partition.yml.erb.

In current form, all three will be part of all pipelines and render code will fail.